### PR TITLE
re-enable jinja2 bytecode caching

### PR DESCRIPTION
**Pull request by fanjinfei**
Fixes #

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by fanjinfei** [comment]
jinja2 caching was disable due to translation issue and it is fixed in https://github.com/open-data/ckanext-canada/pull/903.

**Comment by TkTech** [comment]
Fantastic!

**Comment by wardi** [comment]
Is this something that should be available in core too?

File: ckan/config/environment.py [hunk 1]
```diff
@@ -367,6 +367,9 @@ def template_loaded(template):
         template_paths, auto_reload=True, callback=template_loaded)
 
     # Create Jinja2 environment
+    cache_dir = config.get('jinja2_cache_dir', '/tmp/jinja_cache')
```
Comment (TkTech) [inline ckan/config/environment.py]: See the `tempfile` module (https://docs.python.org/2/library/tempfile.html) to make a secure temporary directory as the default.

**Comment by TkTech** [comment]
@wardi yes, should go in core along with 6cb94a, 762d5c3, 3ae4274 which are all related

**Pull request merged**